src/System.Text.Json/tests/JsonNumberTestData.cs

```diff
@@ -187,13 +187,16 @@ static JsonNumberTestData()
             {
                 // Use InvariantCulture to format the numbers to make sure they retain the decimal point '.'
                 builder.Append("\"double").Append(i).Append("\": ");
-                var str = string.Format(CultureInfo.InvariantCulture, "{0}, ", Doubles[i]);
+                const string format = "{0:" + JsonTestHelper.DoubleFormatString + "}, ";
+                var str = string.Format(CultureInfo.InvariantCulture, format, Doubles[i]);
```

Comment:
The existing code was already using `var`.

Reply:
Yeah, but the Boy Scout Rule (always leave the campground cleaner than you found it).

Reply:
> Yeah, but the Boy Scout Rule (always leave the campground cleaner than you found it).

That depends on whether you like or dislike `var`.

In this case, `var` was already in use and was being used in a place where the type could be considered apparent, which fits both within the overarching style guidelines and the guidelines of keeping the existing styling where possible 😄

Reply:
The general guideline is "apparent" means only new, as-cast and hard-cast.  No factories, no "obviously, everyone knows string.Format returns a string".

Reply:
I would use var here as it's apparent that String.Format returns a string.

Reply:
I agree with Jeremy.  Our rule is basically if the RHS is an explicit cast or `new`, var is fine, otherwise use the type. `string` is immediately understandable, whereas it takes a bit longer to reason out the type of `var`, and it's almost exactly the same length, so it's not a burden to express or read.

Reply:
I've fixed this, but I strongly don't agree with it on two points:
1. The code was already using var. I should not be responsible for fixing up code styling issues that were already reviewed, approved and checked in (this goes against other guidelines we have).
2. We should really consider if there should be a difference between what the compiler can rationalize as apparent and what a dev can rationalize as apparent. If, as a dev, I can't rationalize that `string.Format` returns a string or that `int.Parse` returns an `int` then we have a problem with the APIs we are shipping.